frontend/storage/utils/fetchGameData.ts

```diff
@@ -0,0 +1,65 @@
+import { GameSeries } from '../models/GameSeries';
+import { Game } from '../models/Game';
+import { ApiRequests } from './apiRequestManager';
+import { GameEnvironment } from '../models/GameEnvironment';
+import { AbilityExample } from '../models/AbilityExample';
+import { Ability } from '../models/Ability';
+import { AbilityGroup } from '../models/AbilityGroup';
+import { AbilityCategory } from '../models/AbilityCategory';
+import { createGameAbilitiesListData } from './createGameAbilitiesListData';
+
+export async function fetchGameViewData(id: number) {
+	try {
+		const game: Game = await ApiRequests.get(`games/${id}`, {});
+		const series: GameSeries = await ApiRequests.get(`game-series/${game.seriesId}`, {});
+		return { game, series };
+	} catch (error) {
+		throw new Error(error);
+	}
+}
+
+export async function fetchEnvironments(gameId: number) {
+	try {
+		const gameEnvironments: Array<GameEnvironment> = await ApiRequests.get(`game-environments`, { gameId: gameId });
+		return await ApiRequests.get('environments/', { id: unique(gameEnvironments.map(x => x.environmentId)) });
+	} catch (error) {
+		throw new Error(error);
+	}
+}
+export async function fetchGameBoxArt(gameId: number) {
+	try {
+		const game: Game = await ApiRequests.get(`games/${gameId}`, {});
+		if (game.imageId) {
+			return await ApiRequests.get(`images/${game.imageId}`, {});
+		} else {
+			return null;
+		}
+	} catch (error) {
+		throw new Error(error);
+	}
+}
+export async function fetchGameAbilitiesListData(gameId: number) {
+	try {
+		const abilityExamples: Array<AbilityExample> = await ApiRequests.get(`ability-examples`, { gameId: gameId });
```

Comment:
To simplify this code a bit more, in `ApiRequests.get` add generic param that will set the return type, so these lines look like:
```
const abilityExamples = await ApiRequests.get<AbilityExample>(...)
```